ymer/src/main/java/com/avanza/ymer/PersistInstanceIdDefinition.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015 Avanza Bank AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.avanza.ymer;
+
+public final class PersistInstanceIdDefinition<T> {
+
+	private static final int DEFAULT_DELAY = 60 * 60;
+
+	private final MirroredObjectDefinition<T> parent;
+	private boolean enabled;
+	private boolean recalculateOnStartup;
+	private int recalculateWithDelay = DEFAULT_DELAY;
```

Comment:
Maybe `recalculateWithDelay` should be a `Duration` rather than an `int`?

Reply:
Sure, I changed it to a `Duration`. The override property is still defined in seconds.